Library/ViewModels/PledgePaymentMethodsViewModelTests.swift

```diff
@@ -652,13 +678,15 @@ final class PledgePaymentMethodsViewModelTests: TestCase {
   func testTrackingEvents() {
     let project = Project.template
 
-    self.vm.inputs.configureWith((User.template, project, true))
     self.vm.inputs.viewDidLoad()
+    self.vm.inputs.configure(with: (User.template, project, Reward.template))
 
     XCTAssertEqual([], self.trackingClient.events)
 
     self.vm.inputs.addNewCardTapped(with: .pledge)
 
+    let props = self.trackingClient.properties.last
```

Comment:
This variable is not being used. Did you write this to test the properties sent with the event?

Reply:
Hm so the properties are tested in `KoalaTests.swift`. I can remove this line 👍